src/main/java/org/openlmis/cce/repository/InventoryItemRepository.java

```diff
@@ -49,4 +52,10 @@ Boolean existsByEquipmentTrackingIdAndCatalogItem_ModelAndCatalogItem_Type(
       + " ORDER BY ?#{#pageable}",
       nativeQuery = true)
   Page<InventoryItem> findAllWithoutSnapshots(Pageable pageable);
+
+  @Query("SELECT SUM(i.catalogItem.netVolume) FROM InventoryItem i "
+          + "WHERE i.facilityId = :facilityId AND i.functionalStatus = :functionalStatus")
+  Optional<Number> getInventoryItemVolume(
```

Comment:
I think we could rename it to "getFacilityFunctioningVolume" and remove functionalStatus parameter and make it always FUNCTIONING.